statescript/executor.go

```diff
@@ -38,6 +49,26 @@ type Launcher struct {
 	RootfsScriptsPath       string
 	SupportedScriptVersions []int
 	Timeout                 int
+	RetryInterval           int
+	RetryTimeout            int
+}
+
+func (l *Launcher) getRetryInterval() time.Duration {
+
+	if l.RetryInterval != 0 {
+		return time.Duration(l.RetryInterval) * time.Second
+	}
+	log.Warningf("No timeout interval set for the retry-scripts. Falling back to default: %2s", defaultRetryTimeout.String())
+	return defaultRetryTimeout
```

Comment:
It is printed as a string so that I can limit the width of the print, for pretty-printing.